db/batchlog_manager.cc

```diff
@@ -290,7 +290,7 @@ future<> db::batchlog_manager::replay_all_failed_batches() {
             mutation m(schema, key);
             auto now = service::client_state(service::client_state::internal_tag()).get_timestamp();
             m.partition().apply_delete(*schema, clustering_key_prefix::make_empty(), tombstone(now, gc_clock::now()));
-            return _qp.proxy().mutate_locally(m);
+            return _qp.proxy().mutate_locally(m, tracing::trace_state_ptr());
```

Comment:
Please, put a TODO here to instrument logged batches so that if the original statement was traced we would would hold the trace_state object up until the statement was successfully replayed.

Reply:
I don't think this can be implemented, we'd need to store the trace state in the hint.

Reply:
@avikivity Storing a session_id (UUID) would be sufficient. On a replay a new trace span for the corresponding session_id will be created.

Reply:
We can't trace the replay path of batch logs, for the same reason (we should be able to trace the foreground path)

Reply:
@avikivity There isn't any technical difficulty to trace the replay path as I described above: all we need is to store a `session_id` value with the corresponding batch statement.